js-core/homeworks/homework-2/src/main.js

```diff
@@ -0,0 +1,79 @@
+/*
+ 1. Составить предложение из нижестоящих переменных:
+ "Сколько нужно программистов чтобы сделать проект ? 1, 25, команда"
+ */
+
+let numbers = [25, 1];
+let project = 'проект';
+let team = `команда`;
+let howMuch = 'Сколько';
+let sentence = 'нужно программистов чтобы сделать проект ?';
+
+// --- Решение
+
+// let task1 = howMuch + sentence + ' ' + numbers[1] + ', ' + numbers[0];
+let task1 = `${howMuch} ${sentence}  ${numbers[1]}, ${numbers[0]}`;
+
+
+console.log(task1);
+
+
+/*
+ 2. Составьте предложение из представленного массива
+ и выведите предложение в консоль;
+ "Так, когда Будда достиг Просветления, он обнаружил, что больше не ощущает себя мишенью.
+  Он не был больше ни телом, к которому рано или поздно ..."
+  // index +++
+ */
+
+let array = [
+    'Он',
+    'был больше ни телом, к которому рано или поздно',
+    'он обнаружил',
+    'не',
+    'Так, когда Будда достиг Просветления',
+    '...',
+    'что больше',
+    'ощущает себя мишенью',
+    'не'
+  ];
+
+  // --- Решение
+  
+  let homeSentence = '"' + array[4] + ', ' + array[2] + ', ' + array[6] + ' ' + array[8] + 
+  array[7] + '.' + '\n' + array[0] + ' ' + array[3] + ' ' + array[1] + ' ' + array[5] + '"'
+
+console.log(homeSentence);
+
+
+/*
+ 3. Добавьте свойста 4 новых свойства в объект используя
+    квадратные скобки и точку.
+ */
+
+
+// --- Решение
+
+let myObj = {
```

Comment:
it's not really a solution.

you have to add properties to object via dot, and [].

so you don't need to change initial object

```js
let myObj = {}
```

[objects](https://learn.javascript.ru/object#%D0%B4%D0%BE%D1%81%D1%82%D1%83%D0%BF-%D0%BA-%D1%81%D0%B2%D0%BE%D0%B9%D1%81%D1%82%D0%B2%D1%83-%D1%87%D0%B5%D1%80%D0%B5%D0%B7-%D0%BF%D0%B5%D1%80%D0%B5%D0%BC%D0%B5%D0%BD%D0%BD%D1%83%D1%8E)